helm/robusta/values.yaml

```diff
@@ -76,323 +76,496 @@ disableCloudRouting: false
-  - pod_issue_investigator: {}
-
-- triggers:
-  - on_prometheus_alert:
-      alert_name: KubeStatefulSetUpdateNotRolledOut
-  actions:
-  - related_pods: {}
-  - statefulset_replicas_enricher: {}
-
-
-# Prometheus Daemonset playbooks
-- triggers:
-  - on_prometheus_alert:
-      alert_name: KubeDaemonSetRolloutStuck
-  actions:
-  - resource_events_enricher: {}
-  - related_pods: {}
-  - daemonset_status_enricher: {}
-
-- triggers:
-  - on_prometheus_alert:
-      alert_name: KubernetesDaemonsetMisscheduled
-  - on_prometheus_alert:
-      alert_name: KubeDaemonSetMisScheduled
-  actions:
-  - daemonset_status_enricher: {}
-  - daemonset_misscheduled_analysis_enricher: {}
-
-- triggers:
-  - on_prometheus_alert:
-      alert_name: HostHighCpuLoad
-  actions:
-  - node_cpu_enricher: {}
-  - alert_graph_enricher:
-      resource_type: CPU
-      item_type: Node
-- triggers:
-  - on_prometheus_alert:
-      alert_name: HostOomKillDetected
-  actions:
-  - oom_killer_enricher: {}
-  - alert_graph_enricher:
-      resource_type: Memory
-      item_type: Node
-- triggers:
-  - on_prometheus_alert:
-      alert_name: NodeFilesystemSpaceFillingUp
-  - on_prometheus_alert:
-      alert_name: NodeFilesystemAlmostOutOfSpace
-  actions:
-  - node_disk_analyzer: {}
-  - alert_graph_enricher:
-      resource_type: Disk
-      item_type: Node
-- triggers:
-  - on_prometheus_alert:
-      alert_name: CPUThrottlingHigh
-      status: "all" # sometimes this enricher silences the alert, so we need to silence it regardless of status
-  actions:
-  - cpu_throttling_analysis_enricher: {}
-  - alert_graph_enricher:
-      resource_type: CPU
-      item_type: Pod
-- triggers:
-  - on_prometheus_alert:
-      alert_name: KubernetesDeploymentReplicasMismatch
-  - on_prometheus_alert:
-      alert_name: KubeDeploymentReplicasMismatch
-  actions:
-  - pod_issue_investigator: {}
-  - deployment_events_enricher:
-      included_types: ["Warning"]
-  - deployment_events_enricher:
-      included_types: ["Warning", "Normal"]
-      dependent_pod_mode: true
-
-- triggers:
-  - on_prometheus_alert:
-      status: "all"
-  actions:
-  - default_enricher: {}
-
-- triggers:
-  - on_prometheus_alert:
-      alert_name: NodeFilesystemSpaceFillingUp
-      k8s_providers: ["Minikube", "Kind", "RancherDesktop"]
-  - on_prometheus_alert:
-      alert_name: NodeFilesystemAlmostOutOfSpace
-      k8s_providers: ["Minikube", "Kind", "RancherDesktop"]
-  actions:
-  - alert_explanation_enricher:
-      alert_explanation: "This alert is fired when the file system is running out of space."
-      recommended_resolution: "This is a common issue on local clusters and we recommend increasing the node disk size for your cluster to run optimally."
+
+
+
+
+
+###################################
+# playbook map, playbooksMap
+###################################
+playbooksMap:
+  # priority builtin playbooks for running before all playbooks
+  #don't remove it. used for internal processing !!!  !
```

Comment:
I moved this playbook from code(config_loader.py). Removing it will influence normal code execution.
```
active_playbooks = [
            PlaybookDefinition(
                triggers=[{"on_kubernetes_any_resource_all_changes": {}}],
                actions=[{"cluster_discovery_updates": {}}],
            )
        ]
```

Reply:
ohh got it.
Maybe the comment should be:
`This playbook required for Robusta to function properly. Do not disable or remove it.`